modules/platform/nuxeo-platform-filemanager-core/src/main/java/org/nuxeo/ecm/platform/filemanager/service/extension/CSVZipImporter.java

```diff
@@ -111,47 +118,47 @@ public DocumentModel createOrUpdate(FileImporterContext context) throws IOExcept
 
             ZipEntry index = zip.getEntry(MARKER);
             try (Reader reader = new InputStreamReader(zip.getInputStream(index));
-                 CSVParser csvParser = new CSVParser(reader, CSVFormat.DEFAULT.withHeader());) {
+                CSVParser csvParser = new CSVParser(reader, CSVFormat.DEFAULT.withHeader());) {
 
                 Map<String, Integer> header = csvParser.getHeaderMap();
                 for (CSVRecord csvRecord : csvParser) {
                     String type = null;
-                    String id = null;
+                    String name = null;
                     Map<String, String> stringValues = new HashMap<>();
                     for (String headerValue : header.keySet()) {
                         String lineValue = csvRecord.get(headerValue);
                         if ("type".equalsIgnoreCase(headerValue)) {
                             type = lineValue;
-                        } else if ("id".equalsIgnoreCase(headerValue)) {
-                            id = lineValue;
+                        } else if ("name".equalsIgnoreCase(headerValue)) {
+                            name = lineValue;
                         } else {
                             stringValues.put(headerValue, lineValue);
                         }
                     }
 
+                    if (StringUtils.isBlank(name)) {
+                        log.warn("Can not create or update doc without a name, skipping line: {}",
+                                csvRecord.getRecordNumber());
+                        continue;
+                    }
+
                     boolean updateDoc = false;
                     // get doc for update
                     DocumentModel targetDoc = null;
-                    if (id != null) {
-                        // update ?
-                        String targetPath = new Path(parentPath).append(id).toString();
-                        if (session.exists(new PathRef(targetPath))) {
-                            targetDoc = session.getDocument(new PathRef(targetPath));
-                            updateDoc = true;
-                        }
+
+                    String targetPath = new Path(parentPath).append(name).toString();
+                    if (session.exists(new PathRef(targetPath))) {
+                        targetDoc = session.getDocument(new PathRef(targetPath));
+                        updateDoc = true;
                     }
 
                     // create doc if needed
                     if (targetDoc == null) {
                         if (type == null) {
-                            log.error("Can not create doc without a type, skipping line");
+                            log.error("Can not create doc without a type, skipping line: {}", name);
```

Comment:
I had to double check because the other "skipping line" message above logs the line number, whereas here we log the line "name" but this is a good enough identifier. Although I guess it wouldn't hurt to also add the line number.

Reply:
I replaced with line number for simplicity and also changed to warn instead of error